documentation/code-gen/configure-python-sdk.md

```diff
@@ -0,0 +1,229 @@
+
+This block is updated by an automatic script. Edits may be lost!
+
+``` yaml $(tag) == 'all-api-versions' /* autogenerated */
+# include the azure profile definitions from the standard location
+require: $(this-folder)/../../../profiles/readme.md
+
+# all the input files across all versions
+input-file:
+  - Microsoft.Compute/stable/2019-12-01/compute.json
+  - Microsoft.Compute/stable/2019-12-01/runCommands.json
+  - Microsoft.Compute/stable/2019-12-01/gallery.json
+  - ...
+~~~~
+
+
+## Swagger to SDK
+To make python SDK can be generated from the tag, swagger-to-sdk need to be configured:
+
+~~~
+// file: readme.md
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - ...
+
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+~~~
+
+## Python Configuration
+Python dedicated configurations are configured in readme.python.md.
+A typical readme.python.md is like this:
+
+~~~
+// file: readme.python.md
+
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml !$(track2)                         // For track1: basic Python package information 
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.appconfiguration
+  package-name: azure-mgmt-appconfiguration
+  package-version: 0.1.0
+  clear-output-folder: true
+```
+
+These settings apply only when `--track2` is specified on the command line.
+
+``` yaml $(track2)                          // For track2: basic Python package information 
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+package-name: azure-mgmt-appconfiguration
+no-namespace-folders: true
+package-version: 0.1.0
+```
+
+``` yaml $(python-mode) == 'update'
+// --------------- For track1 -----------------
+python:                                     
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
+
+// --------------- For track2 -----------------
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
+```
+``` yaml $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+basic-setup-py: true
+output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+```
+~~~
+
+## Multi-api
+If a multi-api package is need to be released, 'batch' should be used in the readme.python.md.
```

Comment:
Maybe explain a little bit about multi-api to the service team. Basically it's a package that contains the API versions?